contracts/Lockbox.cdc

```diff
@@ -0,0 +1,468 @@
+/*
+
+    Lockbox implements the functionality required to manage FLOW
+    buyers locked tokens from the token sale.
+
+    Each token holer gets two accounts. One account is their locked token
+    account. It will be jointly controlled by the user and Dapper Labs.
+    No actions will be able to be performed with this account without authorization
+    from Dapper Labs. Locked tokens are stored in this account and Dapper Labs
```

Comment:
```suggestion
    No actions can be performed by this account without authorization
    from both the token holder and the token administrator. 
```

The current phrasing sort of implies that the token admin can move funds without approval from the token holder -- might be better to be extra explicit here.